scripts/util/dont-show-again.ts

```diff
@@ -1,31 +1,28 @@
-const PS_OJBECT_KEY = 'dosa';
+const PS_OBJECT_KEY = 'dosa';
 
 /**
  * A utility of handling anything with DOn't Show Again behaviour (not rice-based pancakes).
  */
 class DosaHandler {
 	/**
-	 * Store a DoSA.
-	 * @param {String} key
-	 * @param {String} nameToken
+	 * Store a DoSA
 	 */
-	static addDosa(key, nameToken) {
-		const dosas = this.#getPSObject();
+	static addDosa(key: string, nameToken: string) {
+		const dosas = this.getPSObject();
 		dosas[key] = nameToken;
-		this.#setPSObject(dosas);
+		this.setPSObject(dosas);
 	}
 
 	/**
-	 * Remove a stored DoSA.
-	 * @param {String} key
-	 * @returns {Boolean} Whether the DoSA was deleted
+	 * Remove a stored DoSA
 	 */
-	static removeDosa(key) {
-		const dosas = this.#getPSObject();
-
+	static removeDosa(key: string): boolean {
+		const dosas = this.getPSObject();
+    if (!dosas) return false;
```

Comment:
Spaces? Does prettier enforce tabs vs spaces at all?

Reply:
Was baffled as to why Prettier wasn't catching this in CI then realized I hadn't configured it to test `.ts` files in `package.json`, fixed now!